content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,24 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-rc.2 [2020-10-22]
+
+### Features
+
+- Upgrade to [Flux v0.90.0](/influxdb/v2.0/reference/release-notes/flux/#v0-90-0-2020-10-19).
+- Add `--force` option to the `influx stacks rm` command, which lets you remove a stack without the confirmation prompt.
+- Add `aggregate_resultset` for mean aggregate pushdown to optimize window results.
```

Comment:
is "optimize window results" correct?

Reply:
optimize windowed results i think would be more accurate